core/experiments/manager.php

```diff
@@ -410,6 +410,12 @@ private function add_default_features() {
 			), '<a target="_blank" href="https://go.elementor.com/wp-dash-flex-container/">', '</a>', '<a target="_blank" href="https://go.elementor.com/wp-dash-flex-container-playground/">', '</a>'),
 			'release_status' => self::RELEASE_STATUS_RC,
 			'default' => self::STATE_INACTIVE,
+			'messages' => [
+				'on_deactivate' => esc_html__(
+					'you will permanently delete all content created with containers and lose access to container-based features like Tabs and Menu',
```

Comment:
maybe just put the whole text here?
("if you deactivate ...")

Reply:
Done